Container/exercise06/app/frontend/client/src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+import React from "react"
+
+function App() {
+
+
+  const [data, setData] = React.useState(null);
+
+  React.useEffect(() => {
+    fetch("/api")
```

Comment:
How will this work? If you've not provided the backend URL,
Read this url using ENV in DockerCompose file